packages/dam-app-base/README.md

```diff
@@ -0,0 +1,57 @@
+[![CircleCI](https://circleci.com/gh/contentful/apps.svg?style=svg&circle-token=913f0d4852062fbed644fca927d059d5e3e72908)](https://circleci.com/gh/contentful/apps)
+
+# dam-app-base
+
+`@contentful/dam-app-base` is a library that helps you to quickly build an app to integrate your DAM (Digital Asset Management) system of choice with Contentful. Many DAM integrations are very similar, therefore this library provides the boilerplate for your app and you only need to add the code for your specific service.
+
+## Usage
+
+```javascript
+import { setup } from '@contentful/dam-app-base';
+
+
+setup({
+  cta: 'Select assets',
+  name: 'My DAM App',
+  logo: 'https://example.com/logo.svg',
+  color: '#d7f0fa',
+  description: 'My example DAM App',
+  parameterDefinitions: [
+    {
+      "id": "folder",
+      "type": "Symbol",
+      "name": "Folder",
+      "description": "Preselected folder when selecting assets.",
+      "required": true
+    }
+  ],
+  validateParameters: () => null,
+  makeThumbnail,
+  renderDialog: (sdk) => {
+    const config = sdk.parameters.invocation;
+
+    const container = document.createElement('div');
+    container.innerHTML = `<iframe src="https://example.com/dam?folder=${config.folder}" />`;
+    document.body.appendChild(container);
+  },
+  openDialog: (sdk, currentValue, config) => {
+    return await sdk.dialogs.openCurrentApp({
+      parameters: { config, currentValue },
+    });
```

Comment:
You cannot away without `async`. Moreover, I think it's clearer to just return the `Promise` here

Reply:
I swear I fixed that...
There is a minor difference between returning the promises and adding an extra `async`/`await` layer. With my implementation, you get an extra stack trace line and can debug your things easier. Also in the existing apps in this repo, the function is way longer, so it's likely for users copying this line to extend the function